app/routes/ready-reckoner/select-standard.js

```diff
@@ -0,0 +1,92 @@
+const standards = require('../../services/standards')
```

Comment:
I don't think this is needed, if `paymentRate` is used from the payload (see comments below).